demo/src/routes/docs/[framework]/components/tree/+layout.server.ts

```diff
@@ -0,0 +1,3 @@
+import {getMenu} from '../getMenu';
```

Comment:
Also add the folder `style` just next to `examples` and `playground` (you can almost copy-paste what was done for slider).

However I noticed that there is a small bug to open the **Style** tab...
At [this line](https://github.com/AmadeusITGroup/AgnosUI/blob/main/demo/src/routes/docs/%5Bframework%5D/components/getMenu.ts#L21) you will need to update from 
```typescript
...(componentMetadata.includeStyles ? [{title: 'Styling', key: 'style', path: '/components/slider/style'}] : []),
```
to
```typescript
...(componentMetadata.includeStyles ? [{title: 'Styling', key: 'style', path: `/components/${component}/style`}] : []),
```